commands/local_server_stop.go

```diff
@@ -92,8 +128,26 @@ var localServerStopCmd = &console.Command{
 		if err := g.Wait(); err != nil {
 			return err
 		}
+	}
 
-		ui.Success(fmt.Sprintf("Stopped %d process(es) successfully", running))
-		return nil
-	},
+	ui.Success(fmt.Sprintf("Stopped %d process(es) successfully", running))
+
+	return nil
+}
+
+func localConfiguredAndRunningProjects() (map[string]*projects.ConfiguredProject, error) {
+	proxyProjects, err := pid.ToConfiguredProjects()
```

Comment:
Should be `config.ToConfiguredProjects()` instead, right? (as the current call is exactly the same as the one below)

Reply:
It should be `proxy.ToConfiguredProjects()` (took that logic from `local_server_list.go`). Maybe it should be `config` instead of `proxy`? I haven't quite figured out the principles of proxy projects here yet so I'm feeling my way around a bit :)